### PR TITLE
Getting started guide

### PR DESCRIPTION
**Pull request by lesnitsky**

### CODE REVIEW

**Review by Shastel — APPROVED** [review]

**Comment by DaQuirm** [comment]
That's should do it, very comprehensive, thanks, @R1ZZU 🐱 
It should be noted, however, that cssqd can also be set up using Docker (for development purposes too). 🐳

File: README.md [hunk 1]
```diff
@@ -2,3 +2,35 @@
...
+* open http://localhost:3000/quick-login.html
+* authorize any provider (e.g GitHub)
+* find your user in mongo users collection
+* update `game_master_id` field of gamesession in `gamesessions` collection with your user `_id`
+* kill cssqd service
+* copy gamesession `_id`
+* `npm run dev`
+* open http://localhost:3000/game?id=<gamesession_id>
+* open same url in different browser/incognito window
+* authorize other provider (e.g Twitter)
```
Comment (DaQuirm) [inline README.md]: another

File: README.md [hunk 2]
```diff
@@ -2,3 +2,35 @@
...
+
+* open http://localhost:3000/quick-login.html
+* authorize any provider (e.g GitHub)
+* find your user in mongo users collection
+* update `game_master_id` field of gamesession in `gamesessions` collection with your user `_id`
+* kill cssqd service
+* copy gamesession `_id`
+* `npm run dev`
+* open http://localhost:3000/game?id=<gamesession_id>
+* open same url in different browser/incognito window
```
Comment (DaQuirm) [inline README.md]: _the_ same

File: config/var/dev.cson [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
 	mongo:
-		host: "mongodb://mongo"
```
Comment (DaQuirm) [inline config/var/dev.cson]: Just FYI: originally it was configured to work in a Docker-based dev setup
However "containerless" development seems more convenient, so yeah, let's change it, I guess.

**Review by DaQuirm — APPROVED** [review]

**Pull request merged**